charts/kubernetes-graphql-gateway/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+version: 0.19.27
```

Comment:
Do we want to bump version? I didn't change it yet.

Reply:
yeah lets probably bump a patch version so that the main pipe can push